redisgears_v8_plugin/src/v8_backend.rs

```diff
@@ -287,23 +270,16 @@ impl BackendCtxInterfaceInitialised for V8Backend {
 
                 let api_version_supported: ApiVersionSupported = api_version.try_into()?;
 
-                let messages = api_version_supported
+                api_version_supported
                     .validate_code(code)
                     .iter()
                     .enumerate()
                     .map(|(index, error)| format!("\t{}. {}", index + 1, error.get_msg()))
-                    .collect::<Vec<String>>()
-                    .join("\n");
-
-                if !messages.is_empty() {
-                    let compiled_message = &format!(
-                        "The module \"{module_name}\" compilation got these messages:\n{}",
-                        messages
-                    );
-
-                    log(compiled_message);
-                    script_ctx.compiled_library_api.log(compiled_message);
-                }
+                    .for_each(|message| {
+                        script_ctx
+                            .compiled_library_api
+                            .log(&format!("Module \"{module_name}\": {message}"))
```

Comment:
If module_name is the library name then I believe `compiled_library_api.log` will add it automatically no?